cpp/log/cert_test.cc

```diff
@@ -85,6 +85,40 @@ static const char kInvalidCertString[] =
 
 static const char kMismatchingSigAlgsCertString[] =
```

Comment:
I added some comments but it made more sense to me to say how kMismatchingSigAlgsCertString differs from kMatchingSigAlgsCertString and how kIllegalSigAlgParameterCertString differs from kMismatchingSigAlgsCertString... which is similar to but not exactly what you asked for. LMK what you think.